samples/WebApi/README.md

```diff
@@ -0,0 +1,113 @@
+# Description
+This is a sample application, which uses CoreRT to compile a .NET Core Web API sample.
+TODO WIP - Might not work on Linix
+
+# Building a WebAPI app with CoreRT
+
+## Install the .NET Core SDK
+CoreRT is an AOT-optimized .NET Core runtime. If you're new to .NET Core make sure to visit the [official starting page](http://dotnet.github.io). It will guide you through installing pre-requisites and building your first app.
+If you're already familiar with .NET Core make sure you've [downloaded and installed the .NET Core 2 SDK](https://www.microsoft.com/net/download/core).
+
+## Create your app 
+
+Open a **new** shell/command prompt window and run the following commands.
+``> dotnet new webapi -o myApp`` 
+
+``> cd myApp``
+
+## Add CoreRT to your project
+
+Using CoreRT to compile your application is done via the ILCompiler NuGet package.
+
+In your shell/command prompt navigate to the root directory of your project run the command 
+
+`` dotnet new nuget ``
+
+This will add a nuget.config file to your application. Open the file and in the ``<packageSources> `` element under ``<clear/>`` add the following:
+
+``<add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" /> ``
```

Comment:
Both lines render on the same line (check https://github.com/A-And/corert/blob/e869d3d93e94f725d33a565988050ba5d7055cc4/samples/WebApi/README.md). You may need to use triple backtick.